app/services/revocation_registry.py

```diff
@@ -487,3 +503,38 @@ async def wait_for_active_registry(
         sleep_duration = 0.5  # Following sleeps should wait 0.5s before retry
 
     return active_registries
+
+
+async def get_pending_revocations(
+    controller: AcaPyClient, rev_reg_id: str
+) -> List[int]:
+    """
+    Get the pending revocations for a revocation registry.
+
+    Args:
+        controller (AcaPyClient): aca-py client
+        rev_reg_id (str): The revocation registry ID.
+
+    Raises:
+        Exception: When the pending revocations could not be retrieved.
+
+    Returns:
+        pending_revocations (List[str]): The pending revocations.
```

Comment:
Mismatch - method says returns List[int], description says List[str]

Also make it more clear what "The pending revocations" is - it's cred_rev_ids?

Looks like `result.result.pending_pub` returns List[str]. Should they be transformed to ints?